### PR TITLE
Support custom data for Raygun Handler

### PR DESCRIPTION
**Pull request by Sazpaimon**
Raygun allows sending extra data to the client. This PR allows you to send that data to Raygun using the appropriate keys.

### CODE REVIEW

**Comment by wpillar** [comment]
Hi @Sazpaimon, thanks for the contribution! Just a few comments from me and then we can try and get this merged.

File: src/Graze/Monolog/Handler/RaygunHandler.php [hunk 1]
```diff
@@ -42,11 +42,21 @@ public function __construct(RaygunClient $client, $level = Logger::DEBUG, $bubbl
     protected function write(array $record)
     {
         $context = $record['context'];
-
+        $tags = $customData = $timestamp = null;
```
Comment (wpillar) [inline src/Graze/Monolog/Handler/RaygunHandler.php]: Could we assign these one per line please?

File: src/Graze/Monolog/Handler/RaygunHandler.php [hunk 2]
```diff
@@ -55,23 +65,26 @@ protected function write(array $record)
     /**
      * @param array $record
      */
-    protected function writeError(array $record)
+    protected function writeError(array $record, $tags = null, $customData = null, $timestamp = null)
```
Comment (wpillar) [inline src/Graze/Monolog/Handler/RaygunHandler.php]: If I'm reading the RayGun code right, `$tags` and `$customerData` should be an arrays, so let's type-hint those correctly.

File: src/Graze/Monolog/Handler/RaygunHandler.php [hunk 3]
```diff
@@ -55,23 +65,26 @@ protected function write(array $record)
...
+            $customData,
+            $timestamp
         );
     }
 
     /**
      * @param array $record
      */
-    protected function writeException(array $record)
+    protected function writeException(array $record, $tags = null, $customData = null, $timestamp = null)
```
Comment (wpillar) [inline src/Graze/Monolog/Handler/RaygunHandler.php]: If I'm reading the RayGun code right, `$tags` and `$customerData` should be an arrays, so let's type-hint those correctly.

File: src/Graze/Monolog/Handler/RaygunHandler.php (hunk 2 above)
Comment (wpillar) [inline src/Graze/Monolog/Handler/RaygunHandler.php]: Don't forget to update the docblocks!

File: src/Graze/Monolog/Handler/RaygunHandler.php (hunk 3 above)
Comment (wpillar) [inline src/Graze/Monolog/Handler/RaygunHandler.php]: Don't forget to update the docblocks!

**Comment by Sazpaimon** [comment]
Updated. Let me know if I missed anything.

**Comment by Sazpaimon** [comment]
I'm actually going to close this and do it a different way that's compatible with the way other handlers handle custom data. I'll make a new PR shortly.

**Pull request closed without merging**